### PR TITLE
[WIP] Create new mobile client UI

### PR DESCRIPTION
**Pull request by wojta**
https://issues.jboss.org/browse/AEROGEAR-7665

- [x] create mobile client form
- [x] make it modal dialog
- [ ] add validation
- [ ] prepare for backend connection

creationclient

### CODE REVIEW

File: ui/src/components/App.scss [hunk 1]
```diff
@@ -0,0 +1,30 @@
+.platform-item > .platform-item-icon {
```
Comment (wojta) [inline ui/src/components/App.scss]: I think that this is not correct location for the styles. I should move it somewhere else... Not sure where?

File: ui/src/components/create_client/CreateXamarinClient.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+/**
+ * Component for the Android specific create mobile client form.
+ */
+class CreateXamarinClient extends Component {
+    render() {
+        const formFields = [
+            {
+                controlId:'appName',
+                label:'* Application Name',
```
Comment (wojta) [inline ui/src/components/create_client/CreateXamarinClient.js]: Is there some recommended proper way in Patternfly (bootstrap) to mark the field required?

File: ui/src/components/App.scss (hunk 1 above)
Comment (ciaranRoche) [reply ui/src/components/App.scss]: At the moment if we need our own `sass` in a `js` file we add a `scss` file specifically for that `js`

**Comment by wojta** [comment]
Moving to PR https://github.com/aerogear/mobile-client-service/pull/34 to use a local branch.

**Pull request closed without merging**